### ISSUE TITLE
Eigen3 with installed feel++ from apt on project.

### ISSUE THREAD

**Issue by vhuber** [feelpp/feelpp#521]
The one that use Feel++ as an extern library, with something like : 
```
cmake ../chorus
```
will face a problem with Eigen3, as long as `FEELPP_ENABLE_SYSTEM_EIGEN3` is set to false.

We should in the default `CMakeLists.txt` used in projects add juste before the `Find_Package(Feel++)`:
```
OPTION(FEELPP_ENABLE_SYSTEM_EIGEN3 "enable system eigen3 support" ON)
```

**Comment by prudhomm** [comment]
there is a slight change in Eigen 3 that I apply that avoid an assertion that eigen3 triggers in some corner case situation and it crashes the code.
If you use the eigen3 system installation you will probably have it at least up to 3.2.3. I haven't checked if they fixed it in 3.2.4

**Comment by vhuber** [comment]
@prudhomm I have the problem with travis. Have you found the solution ?
